### ISSUE TITLE
Add CORS support to SCOW server

### ISSUE THREAD

**Issue by lewisloren** [smart-cow/scow#63]

**Comment by brosenberg42** [comment]
Added in commit 86d63c8a6

**Comment by brosenberg42** [comment]
Example request

``` javascript
$.ajax({
    url: "http://scout2.mitre.org:8080/cow-server/tasks",
    dataType: "json",
    xhrFields: {
        withCredentials: true
    }
}).done(function(data) {
    console.log(data);
});
```

**Comment by brosenberg42** [comment]
Since there is CORS support the GWT RPC stuff is no longer necessary. Since the cow-webapp client can make requests directly to cow-server, all the RPC stuff does is add more latency. However, we may want to leave the RPC stuff in their until we fix the concurrency issues in #51.
